### PR TITLE
Lesson 4

### PR DESCRIPTION
**Pull request by oksana-liseczkaya**

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/Triphouse-project/style.css" />
```
Comment (ghost) [inline Triphouse-project/index.html]: Wrong link /Triphouse-project/style.css

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```
Comment (ghost) [inline Triphouse-project/index.html:1]: Are you open your page in browser ?
2021-04-08_22h34_28
Please check your code, review will be after changes.

File: Triphouse-project/index.html (hunk 1 above)
Comment (oksana-liseczkaya) [reply Triphouse-project/index.html]: fix link

File: Triphouse-project/index.html (hunk 2 above)
Comment (oksana-liseczkaya) [reply Triphouse-project/index.html:1]: I check my code and fix

**Pull request merged**